vmware-event-router/internal/processor/aws_event_bridge.go

```diff
@@ -176,8 +177,9 @@ func NewAWSEventBridgeProcessor(ctx context.Context, cfg connection.Config, sour
 func (awsEventBridge *awsEventBridgeProcessor) Process(moref types.ManagedObjectReference, baseEvent []types.BaseEvent) error {
 	batchInput, err := awsEventBridge.createPutEventsInput(baseEvent)
 	if err != nil {
-		awsEventBridge.Printf("could not create PutEventsInput for event(s): %v", err)
-		return nil
+		errMsg := fmt.Errorf("could not create PutEventsInput for event(s): %v", err)
+		awsEventBridge.Printf(errMsg.Error())
+		return NewError(ProviderAWS, errMsg).Error()
```

Comment:
Can you please provide an example of a formatted error string response in the comment of this PR?

Reply:
Will do 👍

Reply:
Added unit testing which show the expected output ✔️